src/libopensc/card-myeid.c

```diff
@@ -2054,6 +2054,8 @@ myeid_enc_dec_sym(struct sc_card *card, const u8 *data, size_t datalen,
 		if (cbc)
 			apdu.cla = 0x10;
 
+		if (len < apdu_datalen)
+			LOG_FUNC_RETURN(ctx, SC_ERROR_WRONG_LENGTH);
 		len -= apdu_datalen;
```

Comment:
The coverity is saying this:
```
 >>>     CID 427424:  Integer handling issues  (INTEGER_OVERFLOW)
>>>     Expression "len", which is equal to 18446744073709551600, where "apdu_datalen" is known to be equal to 16, underflows the type that receives it, an unsigned integer 64 bits wide.
```
Both of these are `size_t` so this is confusingly saying that when the `len` would be 0, subtracting the `apdu_datalen` from it would result in underflow. But given the `len >= block_size` in the cycle variable, I would probably waive it as an obvious a false positive in the coverity UI.